app/storybook/documentation/ui-kit/Fallbacks.mdx

```diff
@@ -20,14 +20,14 @@ via `ThemeProvider`.
     function App() {
       return (
         <ThemeProvider
-          loaderFallback={() => <div>Loading...</div>}
+          renderLoader={() => <div>Loading...</div>}
```

Comment:
I think the `render` prefix works a lot better 👍
I'll use it for the DataGrid RFC as well